tests/test_featureprofiletools_value_renamer.py

```diff
@@ -17,14 +19,19 @@
 DIR_WITH_OUTPUT_GOLD_STANDARD_INVENTORIES = DIR_WITH_OUTPUT_GOLD_STANDARD_FILES / "inventories"
 
 
-def test_rename_value_in_profiles_and_inventories():
-    # First change files in tmp up to their default variants
+@pytest.fixture
+def create_value_renamer():
```

Comment:
```suggestion
def value_renamer():
```
fixture should be called with a noun in this case